### PR TITLE
[Feature] Showing an active filters that users selected.

### PR DESCRIPTION
**Pull request by YaroslavOvdii**
@sofiiakvasnevska 

## Issue
https://github.com/Fliplet/fliplet-studio/issues/5641

## Description
Showing an active filters that users selected.

## Screenshots/screencasts
https://share.getcloudapp.com/04uYx5Gb

## Backward compatibility
This change is fully backward compatible.

## Reviewers
@upplabs-alex-levchenko

## Notes
When user clicks on an active filter it will be deselected.

### CODE REVIEW

File: js/layout-javascript/small-card-code.js [hunk 1]
```diff
@@ -124,6 +124,11 @@ DynamicList.prototype.attachObservers = function() {
     }
   });
 
+  _this.Utils.Hooks.activeFilters({
```
Comment (tonytlwu) [inline js/layout-javascript/small-card-code.js:127]:
```suggestion
  if (_this.data.filtersInOverlay) {
    Fliplet.Hooks.on('flListDataAfterRenderList', function toggleActiveFilters() {
      _this.Utils.UI.showActiveOverlayFilters();
    });
  }
```

Comment (YaroslavOvdii) [reply js/layout-javascript/small-card-code.js:127]: @tonytlwu Do I get it right that we need to show active filters only when filter overlay is active?

Comment (tonytlwu) [reply js/layout-javascript/small-card-code.js:127]: @YaroslavOvdii Yes. The proposed feature said, "Show active filters when users apply filters **via the overlay**."

**Comment by YaroslavOvdii** [comment]
Made a new [PR](https://github.com/Fliplet/fliplet-widget-dynamic-lists/pull/423) to solve conflicts.

**Pull request closed without merging**